src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/ContentItemQuery.cs

```diff
@@ -47,7 +48,7 @@ public Task BuildAsync(ISchema schema)
             return Task.CompletedTask;
         }
 
-        private Task<ContentItem> ResolveAsync(ResolveFieldContext context)
+        private Task<ContentItem> ResolveAsync(IResolveFieldContext context)
         {
             var contentItemId = context.GetArgument<string>("contentItemId");
             var contentManager = _httpContextAccessor.HttpContext.RequestServices.GetService<IContentManager>();
```

Comment:
not really sure why this is using _httpContextAccessor; it should probably just use `context.RequestServices`